pages/pools/details.tsx

```diff
@@ -1,8 +1,46 @@
-import { Box } from '@interest-protocol/ui-kit';
 import { NextPage } from 'next';
+import { useEffect, useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
 
-const PoolDetailsPage: NextPage = () => {
-  return <Box>Pool Details Page</Box>;
+import { SEO } from '@/components';
+import { PoolPageProps } from '@/interface';
+import PoolDetails from '@/views/pool-details';
+// import { PoolDetailsProvider } from '@/views/pool-details/pool-details.context';
+import { PoolForm, PoolOption } from '@/views/pools/pools.types';
+
+const PoolDetailsPage: NextPage<PoolPageProps> = () => {
+  const [poolOptionView, setPoolOptionView] = useState<PoolOption>(
+    PoolOption.Deposit
+  );
+
+  const handleOptionTab = (index: PoolOption) => setPoolOptionView(index);
+
+  const form = useForm<PoolForm>({
+    defaultValues: {
+      tokenList: [],
+      settings: {
+        slippage: '0.1',
+      },
+    },
+  });
+
+  useEffect(() => {
+    form.resetField('lpCoin.value');
+    form.resetField('tokenList.0.value');
+    form.resetField('tokenList.1.value');
+  }, [poolOptionView]);
+
+  return (
+    <FormProvider {...form}>
+      {/* <PoolDetailsProvider objectId={objectId}> */}
```

Comment:
Here as well if you're using, I understand you're keeping this for future use but is leaving the code dirty.